### ISSUE TITLE
Doubts about Regularization in policy loss

### ISSUE THREAD

**Issue by Marxvans** [pranz24/pytorch-soft-actor-critic#39]
Thank you for your contribution. However I'm confused about 
`reg_loss = 0.001 * (mean.pow(2).mean() + log_std.pow(2).mean())# Regularization Loss` 
in the code. Can you explain it, any help will be grateful.
